tests/integration_tests/test_vision.py

```diff
@@ -115,13 +119,14 @@ def test_record_video(self):
         imgfile_count_before = len(list(Path(c.BB_VIDEO_PATH).glob("*.jpg")))
 
         # test that video can be recorded
-        url = (
-            f"http://localhost:{c.BB_VISION_PORT}/record_video?duration={TEST_DURATION}"
-        )
-        response = requests.get(url)
+        response = vision_client.send_record_video_request(TEST_DURATION)
         assert response.status_code == 200
         assert response.json()["status"] == "ok"
 
```

Comment:
Maybe should also test that the vision service will return not ok if we try to send a second request while the first is still recording.